src/pages/index.js

```diff
@@ -219,3 +220,44 @@ addProfilePopup.setEventListeners();
 const nameSelector = ".profile__title";
```

Comment:
Needs correction: I don't see the code for editing the profile, the avatar, creating cards